gctscript/modules/gct/gct_test.go

```diff
@@ -0,0 +1,418 @@
+package gct
+
+import (
+	"errors"
+	"os"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/d5/tengo/objects"
+	"github.com/thrasher-corp/gocryptotrader/currency"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/asset"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/order"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/orderbook"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/ticker"
+	"github.com/thrasher-corp/gocryptotrader/gctscript/modules"
+)
+
+var (
+	exch = &objects.String{
+		Value: "BTC Markets",
+	}
+	exchError = &objects.String{
+		Value: "error",
+	}
+	currencyPair = &objects.String{
+		Value: "BTC-AUD",
+	}
+	delimiter = &objects.String{
+		Value: "-",
+	}
+	assetType = &objects.String{
+		Value: "SPOT",
+	}
+	orderID = &objects.String{
+		Value: "1235",
+	}
+
+	tv            = objects.TrueValue
+	fv            = objects.FalseValue
+	errTestFailed = errors.New("test failed")
+)
+
+func TestMain(m *testing.M) {
+	modules.SetModuleWrapper(Wrapper{})
+	os.Exit(m.Run())
+}
+
+func TestExchangeOrderbook(t *testing.T) {
+	t.Parallel()
+	_, err := ExchangeOrderbook(exch, currencyPair, delimiter, assetType)
```

Comment:
Since you're predefining these and running them through the script, it would be good to test that the results match what you pass in